### ISSUE TITLE
Unattended Batch Rendering

### ISSUE THREAD

**Issue by moracenr** [smeighan/xLights#2735]
Is it possible to add the option to bypass the "autosave confirmation modal"  when doing a batch rendering, or at least a checkbox in the sequence selection screen that allows to always say 'Yes' to this confirmation screen, the intention is to globally ignore this particular msg, while still report other rendering issues (report on rendering issues option under preferences)

**Comment by computergeek1507** [comment]
In File->Preferences Uncheck "Prompt Issues During Batch Render"

![image](https://user-images.githubusercontent.com/2293065/128434129-af438d6c-30ef-40df-a1cf-6ad91d830b9e.png)

**Comment by moracenr** [comment]
I would like to leave "Prompt Issues During Batch Render" on, that is what I tried to explain in my last line: "to globally ignore this particular msg, while still report other rendering issues (report on rendering issues option under preferences)". I want to know if there are other rendering issues; but be able to ignore this warning. Is this the only problem reported?

I wonder which is the default answer if you turn off this option. Does it mean, use the autosave file? I started a 30 sequence batch rendering overnight, and this morning, I have that modal on the screen, still on the 3 sequence!!!!Maybe turning off autosave altogether...
